docroot/modules/custom/civictheme_migrate/config/install/migrate_plus.migration.menu_link_content_civictheme_footer.yml

```diff
@@ -1,18 +1,17 @@
-uuid: aeaba3ee-b67d-476d-8dc0-8e856f81bb63
 langcode: en
 status: true
 dependencies:
   enforced:
     module:
       - civictheme_migrate
-id: menu_civictheme_footer
+id: menu_link_content_civictheme_footer
```

Comment:
here and below - the entity is `menu_link_content` and bundle is `civictheme_footer` - renamed migrations and fixture files accordingly